content/en/borealis/Reference/ref-deployment-file.md

```diff
@@ -271,6 +301,17 @@ strategies:
               - <queryName>
         - setWeight:
             weight: <integer>
+  <strategyName>
+    blueGreen:
+      activeService: <active-service>
+      previewService: <preview-service>
+      redirectTrafficAfter:
+        pause:
```

Comment:
- pause: 
      untilApproved: true
The pause should have "-" before it. since this is a list of steps in YAML.